apps/condo/domains/common/utils/date.js

```diff
@@ -36,10 +43,65 @@ const getStartDates = (dateRaw) => {
  */
 const getCurrTimeStamp = () => Math.floor(Date.now() / 1000)
 
+/**
+ * Clear bad user input
+ * @param dateStr raw user input
+ * @returns {null|string}
+ */
+function clearDateStr (dateStr) {
```

Comment:
Naming is a bit unconvincing. 

I expect it to work like that:
```
const clearedInput = clearInput(input)
const isValid = isValidInput(clearedInput)
```

Reply:
`null` in return type is bad, since you cannot predict it from function name. You expect to clear `string`, not to override it with `null`. 

That's bad pattern, which is acceptable in-place (in specific mutation file), but not in common utils